siac/hostcmd.go

```diff
@@ -175,7 +176,7 @@ RPC Stats:
 	Revise Calls:       %v
 	Settings Calls:     %v
 	FormContract Calls: %v
-`, is.NetAddress, nm.ErrorCalls, nm.UnrecognizedCalls, nm.DownloadCalls,
+`, nm.NetAddress, nm.ErrorCalls, nm.UnrecognizedCalls, nm.DownloadCalls,
```

Comment:
too simple, you want the user to be able to know whether they are seeing the auto address or the net address... I think

Perhaps it's enough to display a boolean indicating whether the field has been set manually by the user. `Address: Automatic` vs `Address: Manual`, or maybe `Net Address: 12.34.56.78 (automatic)`.

Tricky to do this in a way that easily conveys what happens if they set it manually vs. set it automatically. Maybe it should be labeled 'static' vs 'dynamic', where 'dynamic' is the auto address, and 'static' is manually configured address.